### DISCUSSION TITLE
can I still do nvm uninstall when using pnpm

### DISCUSSION THREAD

**Discussion by laoshaw** [pnpm/pnpm#4087] [General]
pnpm uses hard link, it seems if I `nvm uninstall old-version` it will remove all the packages installed there, so I have to reinstall everything with pnpm again when I use nvm to upgrade node? which means I will have to keep all node versions under ~/.nvm from now on?

**Comment by zkochan** [comment]
You will probably need to reinstall the globally installed packages. But the global content-addressable store is not under the `~/.nvm` dir, so it won't be removed.

Also, you may use pnpm to manage Node.js versions instead of nvm: https://pnpm.io/cli/env
